tests/test_vertical_portscans.py

```diff
@@ -4,6 +4,24 @@
 import base64
 import os
 from tests.module_factory import ModuleFactory
+from slips_files.common.slips_utils import utils
+import unittest
+
+from slips_files.core.evidence_structure.evidence import \
+    (
+        Evidence,
+        ProfileID,
+        TimeWindow,
+        Attacker,
+        ThreatLevel,
+        EvidenceType,
+        IoCType,
+        Direction,
+        IDEACategory,
+        Victim,
+        Proto,
+        Tag
```

Comment:
You're only using Proto from this import list, why import the following? usually your IDE should warn you about this and can even auto fix them for you so you don't have to worry about it

Reply:
Sorry about that. I am actually using VS Code now, so this feature must be disabled. I will fix this mistake and update the pull request.